### ISSUE TITLE
The address is not a valid memory location

### ISSUE THREAD

**Issue by UnlimitedChild** [x64dbg/x64dbg#2496]
Hi,

snapshot_2020-10-29_00-29.zip

Breakpoint at 0000000000401000 (entry breakpoint) set!
No such breakpoint "LdrInitializeThunk"
DLL Loaded: 00007FFEA0760000 C:\Windows\System32\ntdll.dll
DLL Loaded: 0000000000510000 C:\Windows\System32\kernel32.dll
DLL Loaded: 0000000000750000 C:\Windows\System32\KernelBase.dll
[ScyllaHide] Loaded VA for NtUserBlockInput = 0x00007FFE9DD37F30
[ScyllaHide] Loaded VA for NtUserQueryWindow = 0x00007FFE9DD31290
[ScyllaHide] Loaded VA for NtUserBuildHwndList = 0x00007FFE9DD31410
[ScyllaHide] Loaded VA for NtUserFindWindowEx = 0x00007FFE9DD31E10
[ScyllaHide] Loaded VA for NtUserGetClassName = 0x00007FFE9DD31FB0
[ScyllaHide] Loaded VA for NtUserInternalGetWindowText = 0x00007FFE9DD31CD0
System breakpoint reached!
INT3 breakpoint "entry breakpoint" at <28.vmp.EntryPoint> (0000000000401000)!

```
---------------------------
Invalid address!
---------------------------
The address 00007FFE9DD31410 is not a valid memory location...
---------------------------
OK   
---------------------------
---------------------------
Invalid address!
---------------------------
The address 00007FFE9DD31FB0 is not a valid memory location...
---------------------------
OK   
---------------------------

```

**Comment by mrexodia** [comment]
Is it crashing x64dbg? If so, please upload your minidump.

**Comment by UnlimitedChild** [comment]
> Is it crashing x64dbg? If so, please upload your minidump.

no, in the log it is enough to click on the address, then a message box appears ..
[ScyllaHide] Loaded VA for NtUserBuildHwndList = **0x00007FFE9DD31410**
The address **00007FFE9DD31410** is not a valid memory location...
